forge/test/operators/matmul/test_matmul_fuse_bias.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+import numpy as np
+
+import torch
+from torch import nn
+
+import forge
+from forge.op.eval.common import compare_with_golden_pcc
+
+import os
+
+@pytest.mark.parametrize("dims", [
+    (1, 784), (4, 784), (16, 784), (32, 784), (64, 784)
```

Comment:
One more advice, as 784 dim is appearing in every case, you can make it as separate pytest param, and only iterate over batch param :))

Reply:
done